astro/src/content/docs/extend/events-and-webhooks/events/user-login-failed.mdx

```diff
@@ -47,6 +47,22 @@ export const eventType = 'user.login.failed';
     Moved to <InlineField>event.info</InlineField> in 1.30.0
   </APIField>
 
+  <APIField slot="trailing-fields" name="event.reason" type="Object" since="1.53.0">
+    An object containing data on the reason the login failed if it was prevented by a [Login Validation Lambda](/docs/extend/code/lambdas/login-validation).
```

Comment:
This is always present in the response starting in this version - regardless of the reason for which we are sending the event.